README.md

```diff
@@ -139,6 +139,11 @@ cargo run -- --dump-on exit --dump-path <PATH>
 cargo run -- --dump-on transaction --dump-path <PATH>
 ```
 
+Dumping on request (replace <HOST>, <PORT> and <PATH> with your own):
```

Comment:
Look at other documented cases (Dumping on exit, Dumping after each transaction), they each have a dash in the beginning of the line.

Reply:
Fixed in f0e89403cd85fcc72ccff8087642f628806fe339